types/office-js-preview/index.d.ts

```diff
@@ -13530,14 +13532,14 @@ declare namespace Office {
          *
          * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points | Applicable Outlook mode}**: Compose
          *
-         * @param headers - The names and corresponding values of the headers to be set. Should be a dictionary object with keys being the names of the
-         *                internet headers and values being the values of the internet headers.
+         * @param headers - The names and corresponding values of the headers to be set. This should be a record object with its keys being internet header names
+         *                and values being the equivalent header value strings.
```

Comment:
I think "equivalent" should be "corresponding".

Reply:
Thanks, Ricky. This has now been changed.